packages/form-js-viewer/src/render/components/form-fields/Number.js

```diff
@@ -45,30 +153,46 @@ export default function Number(props) {
       id={ prefixId(id, formId) }
       label={ label }
       required={ required } />
-    <input
-      class="fjs-input"
-      disabled={ disabled }
-      id={ prefixId(id, formId) }
-      onInput={ onChange }
-      type="number"
-      value={ value || '' } />
+    <div class={ classNames('fjs-input-group', { 'disabled': disabled }, { 'hasErrors': errors.length }) }>
+      <input
+        ref={ inputRef }
+        class="fjs-input"
+        disabled={ disabled }
+        id={ prefixId(id, formId) }
+        onKeyDown={ onKeyDown }
+        onKeyPress={ onKeyPress }
+
+        // @ts-ignore
+        onInput={ (e) => setValue(e.target.value) }
+        type="text"
+        autoComplete="off"
+        step={ arrowIncrement }
+        value={ displayValue } />
+      <div class={ classNames('fjs-number-arrow-container', { 'disabled': disabled }) }>
+        <button class="fjs-number-arrowUp" onClick={ () => addIncrement() } tabIndex={ -1 }>˄</button>
```

Comment:
```suggestion
        <button class="fjs-number-arrow-up" onClick={ () => addIncrement() } tabIndex={ -1 }>˄</button>
```

Just to be consistent.